src/monitors/ModuleMonitor.v3

```diff
@@ -360,7 +360,7 @@ private class MemoryAccessProbe(imm: MemArg, size: u64, f: (DynamicLoc, Memory,
 private class MemoryGrowProbe(memory_index: u31, f: (DynamicLoc, Memory, u32) -> Resumption) extends Probe {
 	def fire(dynamicLoc: DynamicLoc) -> Resumption {
 		var accessor = dynamicLoc.frame.getFrameAccessor();
-		var pages = u23.!(Values.v_i(accessor.getOperand(0)));
+		var pages = u32.!(Values.v_i(accessor.getOperand(0)));
```

Comment:
I think you want `Values.v_u` here, because a cast will cause an exception for negative values.